pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -434,7 +434,7 @@ class SETroubleshootPage extends React.Component {
                 }
                 return (
                     <cockpitListing.ListingRow
-                        key={itm.details.localId || index}
+                        key={itm.details && itm.details.localId || index}
```

Comment:
This looks fine in principle, but can you please use `? :` instead of `&& ||` for this? eslint fails on this:

      437:42  error  Unexpected mix of '&&' and '||'  no-mixed-operators